src/metabase/automagic_dashboards/names.clj

```diff
@@ -132,55 +157,46 @@
        :day-of-year
        :week-of-year)  (u.date/extract dt unit))))
 
+(defn- humanize-filter-value-op
+  "Create a humanized comparison string for an item, where the comparison string may have two values based on the
+  item type (temporal or non-temporal). For example 'today is after yesterday' vs. '5 is greater than 4'."
+  ([root [_ field-reference value] temporal-str nontemporal-str]
+   (let [{:keys [item-name effective_type base_type unit]} (item-reference->field root field-reference)]
+     (if (isa? (or effective_type base_type) :type/Temporal)
+       (tru "{0} {1} {2}" item-name temporal-str (humanize-datetime value unit))
```

Comment:
I think we lose translation strings this way. We used to have `"{0} is not before {1}"` and other similar strings as arguments to `tru`. But now they are args to a function call which uses `(tru "{0} {1} {2}")` so our i18n script won't pick these strings up as needing translation

Reply:
background we use this [namespace](https://github.com/metabase/metabase/blob/master/bin/build/src/i18n/enumerate.clj#L66) to look for forms like `(trs "...")` and know we need to translate those strings. The way you have it here violates how we look for them so we won't translate these strings.

Reply:
Got it. Take a look now and I think it will be fixed.